langchain/src/graphs/neo4j_graph.ts

```diff
@@ -58,7 +58,9 @@ export class Neo4jGraph {
       await graph.refreshSchema();
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
     } catch (error: any) {
-      throw new Error(`Error: ${error.message}`);
+      throw new Error(`Could not use APOC procedures.
+      Please ensure the APOC plugin is installed in Neo4j and that
```

Comment:
Is the indent intentional?

Reply:
No, I'm just not experienced with JS :) Do I simply remove the indent?

Reply:
fixed